atomic_reactor/tasks/binary.py

```diff
@@ -106,6 +112,24 @@ class BinaryExitTask(plugin_based.PluginBasedTask[TaskParams]):
         {"name": "sendmail"},
     ]
 
+    def prepare_workflow(self) -> inner.DockerBuildWorkflow:
+        """Fully initialize the workflow instance to be used for running the list of plugins."""
+        workflow = inner.DockerBuildWorkflow(
+            context_dir=self.get_context_dir(),
+            build_dir=self.get_build_dir(),
+            data=self.workflow_data,
+            namespace=self._params.namespace,
+            pipeline_run_name=self._params.pipeline_run_name,
+            source=self._params.source,
+            user_params=self._params.user_params,
+            reactor_config_path=self._params.config_file,
+            # Set what plugins to run and how
+            plugins_conf=self.plugins_conf,
+            keep_plugins_running=self.keep_plugins_running,
+            annotations_result=self._params.annotations_result,
+        )
+        return workflow
```

Comment:
Let's not duplicate the logic. Use `super().prepare_workflow()` to get workflow, modify as needed, return workflow
```suggestion
        workflow = super().prepare_workflow()
        workflow.annotations_result = self._params.annotations_result
        return workflow
```

Reply:
ok